drivers/sensor/ti/ina23x/ina230.c

```diff
@@ -249,14 +288,14 @@ static const struct sensor_driver_api ina230_driver_api = {
 #endif /* CONFIG_INA230_TRIGGER */
 
 #define INA230_DRIVER_INIT(inst)                                                                   \
-	static struct ina230_data drv_data_##inst;                                                 \
+	static struct ina230_data drv_data_##inst = {.high_precision =                             \
```

Comment:
This can be located in the config structure.

Reply:
Moved it there